core-web/pom.xml

```diff
@@ -175,6 +175,7 @@
                             <filtering>false</filtering>
                             <targetPath>dotAdmin</targetPath>
                         </resource>
+
```

Comment:
This file could be removed from the changelist

Reply:
Removed.